subprojects/shipkit/src/main/groovy/org/shipkit/gradle/release/ReleaseNeededTask.java

```diff
@@ -102,21 +101,6 @@ public void setSkipComparePublications(boolean skipComparePublications) {
         this.skipComparePublications = skipComparePublications;
     }
 
-    /**
-     * If the exception should be thrown if the release is not needed.
-     */
-    public boolean isExplosive() {
```

Comment:
since this one is public api, do we have to keep it and mark it deprecated? or can we remove it an create a 2.1.0?

Reply:
We keep it and deprecate it. Can you create a static utility method like "deprecated(String message)" so that when we ship a new major version, we can wrap up all deprecations?